ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -581,6 +581,16 @@ export const policyCriteriaDescriptors: Descriptor[] = [
         canBooleanLogic: true,
         lifecycleStages: ['BUILD', 'DEPLOY', 'RUNTIME'],
     },
+    {
+        label: 'Days Since CVE Was Published',
```

Comment:
Not sure, but if this is the UI, should this be sentence case to match the others? (e.g., "Days since CVE was last published")
<img width="391" alt="Screenshot 2025-03-27 at 10 09 44 AM" src="https://github.com/user-attachments/assets/85b76070-80e9-46a1-b871-fb87d9ef2aeb" />

I'm not sure if what shows up on the screen is this field (label) or the shortname field below, which *is* in sentence case, so please ignore if n/a.

Reply:
I changed this to "Days Since CVE was published"